synapse/handlers/federation.py

```diff
@@ -1204,11 +1204,19 @@ def _handle_queued_pdus(self, room_queue):
 
     @defer.inlineCallbacks
     @log_function
-    def on_make_join_request(self, room_id, user_id):
+    def on_make_join_request(self, origin, room_id, user_id):
```

Comment:
Yes, though all the other federation server request handling (in both `federation` and `federation_server`) use `origin`, so would probably make more sense to change the name of the transaction origin.

Reply:
it would make more sense to kill off the `origin` field in transactions altogether (#3816)...

don't mind if you want to leave it. Just thought that beginning to switch some of these to `sending_server` or something might clarify things.